libr/core/config.c

```diff
@@ -1749,6 +1749,7 @@ R_API int r_core_config_init(RCore *core) {
 	SETPREF("graph.gv.edge", "", "Graphviz edge style. (arrowhead=\"vee\")");
 	SETPREF("graph.gv.graph", "", "Graphviz global style attributes. (bgcolor=white)");
 	SETPREF("graph.gv.current", "false", "Highlight the current node in graphviz graph.");
+	SETPREF("graph.node_shortcuts", "false", "Enables shortcuts for every node.");
```

Comment:
Yea would be nice, any suggestion? graph.nodejmps?
On May 12, 2016 12:57, "Anton Kochkov" <notifications@github.com> wrote:

> In libr/core/config.c
> <https://github.com/radare/radare2/pull/4869#discussion_r63001235>:
>
> > @@ -1749,6 +1749,7 @@ R_API int r_core_config_init(RCore *core) {
> >  	SETPREF("graph.gv.edge", "", "Graphviz edge style. (arrowhead=\"vee\")");
> >  	SETPREF("graph.gv.graph", "", "Graphviz global style attributes. (bgcolor=white)");
> >  	SETPREF("graph.gv.current", "false", "Highlight the current node in graphviz graph.");
> > +	SETPREF("graph.node_shortcuts", "false", "Enables shortcuts for every node.");
>
> what about shorter name?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/radare/radare2/pull/4869/files/20122a23b26f73f2e458a4353a1afbc919a97c37#r63001235>
>